<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>displacepoint</class>
 <widget class="QMainWindow" name="displacepoint">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>位移监测点信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(38, 183, 248)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
    <column>
     <property name="text">
      <string>监测点名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>监测点编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>是否已设置传感器</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="savepushButton">
       <property name="text">
        <string>保存信息</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delepushButton">
       <property name="text">
        <string>删除监测点信息</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>监测点编号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pointnumlineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>监测点名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pointnamelineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addpointpushButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>增加测点信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnpushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setsensorpushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>设置传感器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pointlinepushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>显示测点曲线</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pointnumlineEdit_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
